library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java

```diff
@@ -241,6 +241,24 @@ public void setOnMarkerClickListener(OnMapMarkerClickListener listener) {
   @Override public boolean isMyLocationEnabled() {
     return trackUserLocation;
   }
+  
+  @Override public boolean isZoomControlsEnabled() {
+    // no-op
+    return false;
+  }
+   
+  @Override public boolean isMyLocationButtonEnabled() {
+    // no-op
+    return false;
+  }
+  
+  @Override public void setZoomControlsEnabled(boolean enabled) {
+     // no-op
+  }
+  
+  @Override public void setMyLocationButtonEnabled(boolean enabled) {
+    // no-op
+  }
```

Comment:
I wasn't able to find it but I'm not super familiar with the JS library. Was looking here https://developers.google.com/maps/documentation/javascript/reference. Let me know if I should be looking elsewhere